backend/src/interfaces/user.interface.ts

```diff
@@ -3,4 +3,5 @@ export interface User {
   name: string;
   email: string;
   password: string;
+  isManagement: boolean;
```

Comment:
I have just updated the DB schema to go along with your updates. You might want to pull and wipe your DB to get them applied.
Also a tip if you're ever locked out keep a postman collection around
{
    "name": "conor",
    "email": "conor@gmail.com",
    "password": "conor",
    "isManagement": true
}